content/chainguard/chainguard-images/about/development-containers.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Can we edit this in place? We are not changing the URL of "chainguard-images" and we are trying to limit the redirects we are doing.